kytos/utils/napps.py

```diff
@@ -335,13 +336,25 @@ def create_napp(cls, meta_package=False):
         print('--------------------------------------------------------------')
         print('')
         while not cls.valid_name(username):
-            username = input('Please, insert your NApps Server username: ')
+            try:
+                username = input('Please, insert your NApps Server username: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
+                sys.exit(0)
 
         while not cls.valid_name(napp_name):
-            napp_name = input('Please, insert your NApp name: ')
+            try:
+                napp_name = input('Please, insert your NApp name: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
```

Comment:
If you accept the suggestion, use the variable here.

Reply:
Ah yes. That shouldn't be a problem. Will refractor it as soon as I can!